src/components/AudioAnalysis/SpAnalysisChartSize.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import Slider from '@material-ui/core/Slider';
+
+const useStyles = makeStyles({
+  root: {
+    maxWidth: 400,
+  },
+  input: {
+    width: 120,
+  },
+});
+
+export default function ChartSize({ value, handleSizeSliderChange }: any) {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      <Typography id="input-slider" gutterBottom color="primary">
+        Zoom
+      </Typography>
+      <Grid container spacing={2} alignItems="center">
+        <Grid item xs>
```

Comment:
on my end the graphs feels laggy. it looks amazing but somehow the animations are delaying.
i.e. when changing the zoom

Reply:
Unsure how to resolve this one. Maybe need your expertise on this :)